testing/trino-product-tests/src/main/java/io/trino/tests/product/TestSystemConnector.java

```diff
@@ -13,21 +13,34 @@
  */
 package io.trino.tests.product;
```

Comment:
You might need updates to selectInformationSchemaColumns.result and showTablesSystemMetadata.result

Reply:
Are you referring to `BaseConnectorSmokeTest#testSelectInformationSchemaColumns` or `AbstractTestIntegrationSmokeTest#testSelectInformationSchemaColumns`? Both tests seem to pass.

Reply:
I'm referring to `testing/trino-product-tests/src/main/resources/sql-tests/testcases/system/selectInformationSchemaColumns.result` and `testing/trino-product-tests/src/main/resources/sql-tests/testcases/system/showTablesSystemMetadata.result`.
I think whenever we add a system table, it starts showing up in there (see corresponding .sql files).